src/retasc/product_pages_api.py

```diff
@@ -0,0 +1,146 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+"""
+Production Pages API
+"""
+
+import os
+from functools import lru_cache
+
+from retasc import constants, requests_session
+
+_no_auth_session = requests_session.requests_session()
```

Comment:
Better to move this to `ProductPagesApi`. Ideally if this is parameter in constructor and does not indicate what kind of session it is in name:

```python
    def __init__(self, api_url: str, session: requests.Session):
        self.api_url = api_url
        self.session = session
```

Reply:
ack

Reply:
Do you mean don't use `_no_auth_session = requests_session.requests_session()`?
Or 
```
def __init__(self, api_url: str, session: requests_session.requests_session()):
```